starter_code/public/stylesheets/styles.css

```diff
@@ -5,4 +5,71 @@ dark-blue: #0A2C42;
 grey: rgba(145, 145, 145, 0.481);
 font-family: Arial, Helvetica, sans-serif
 */
-  
\ No newline at end of file
+* {
```

Comment:
Good but try to use the boilerplate from the cheatsheet so that you have a more structured css file.